tensorflow/python/layers/normalization.py

```diff
@@ -692,6 +709,7 @@ def batch_normalization(inputs,
       trainable=trainable,
       virtual_batch_size=virtual_batch_size,
       name=name,
+      dtype=inputs.dtype,
```

Comment:
Change "inputs.dtype" to "inputs.dtype.base_dtype"

Reply:
Done.